.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# Set update schedule for Sub-modules
```

Comment:
Missing license header.

Reply:
Actually I was going to put it by default but I found that the repository of [Skywater](https://github.com/google/skywater-pdk/blob/main/.github/dependabot.yml) hadn't put a license header so I have followed it.